examples/http_https_socks5_and_socks5h_connect_proxy.rs

```diff
@@ -0,0 +1,215 @@
+//! An example to showcase how one can build a proxy that is both a HTTP, HTTPS, SOCKS5 and SOCKS5H Proxy in one.
+//!
+//! # Run the example
+//!
+//! ```sh
+//! cargo run --example http_https_socks5_and_socks5h_connect_proxy --features=dns,socks5,http-full,borinng
+//! ```
+//!
+//! # Expected output
+//!
+//! The server will start and listen on `:62023`. You can use `curl` to interact with the service:
```

Comment:
```suggestion
//! The server will start and listen on `:62029`. You can use `curl` to interact with the service:
```

You'll need to use a port that is not yet used by another example. Reason we take static ports is to make them easily testable as well. But to avoid conflicts with multiple examples running concurrently they have to have all a unique port. Port `62023` was already taken.